### PR TITLE
Feature/contact info

### PR DESCRIPTION
**Pull request by hehfg**
Added the rest of contact info to footer and made the email in test a lighter blue to stick out from background.

### CODE REVIEW

File: templates/default/footer.tpl.html [hunk 1]
```diff
@@ -1,3 +1,9 @@
 <footer>
-    <a href="mailto:<%= contact %>"><%=contact%></a>
-</footer>
\ No newline at end of file
+        <div>
+                <%= town %>
+                <%= address %>
+            </div>
```
Comment (filiphazardous) [inline templates/default/footer.tpl.html]: Fix indentation for above lines. Use standard format for street address (street name above city - and something to separate them). You seem to have forgotten to check what this looks like in a browser

File: templates/default/footer.tpl.html [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+    <a href="mailto:<%= email %>" class="email"><%= email %></a>
+    <!--Link through phone number?-->
+    <a><%= phone %></a>
```
Comment (filiphazardous) [inline templates/default/footer.tpl.html]: It's nice that you turn the phone number into a link. But please do it properly.

Check: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

**Review by filiphazardous — CHANGES_REQUESTED** [review]

**Comment by hehfg** [comment]
Made the changes you asked for.

File: templates/pcab/assets/css/pcab.css [hunk 3]
```diff
@@ -20,6 +20,10 @@ section {
     margin-right: 10px;
 }
 
-.section-text {
+.contactInfo {
+    list-style-type: none;
 }
 
+.contactlink {
```
Comment (filiphazardous) [inline templates/pcab/assets/css/pcab.css]: Separate words with a dash (-)

File: templates/pcab/assets/css/pcab.css [hunk 4]
```diff
@@ -20,6 +20,10 @@ section {
     margin-right: 10px;
 }
 
-.section-text {
+.contactInfo {
```
Comment (filiphazardous) [inline templates/pcab/assets/css/pcab.css]: Separate words with a dash (-)

**Review by filiphazardous — CHANGES_REQUESTED** [review]

File: templates/default/footer.tpl.html [hunk 5]
```diff
@@ -1,3 +1,8 @@
 <footer>
-    <a href="mailto:<%= contact %>"><%=contact%></a>
-</footer>
\ No newline at end of file
+        <ul class="contactInfo">
+                <li><div><%= address %></div></li>
+                <li><div><%= town %></div></li>
+                <li><a href="mailto:<%= email %>" class="contactlink"><%= email %></a></li>
+                <li><a href="tel:+<%= phone %>" class="contactlink"><%= phone %></a></li>
```
Comment (filiphazardous) [inline templates/default/footer.tpl.html]: Remove the `+` sign - otherwise good

**Review by filiphazardous — CHANGES_REQUESTED** [review]

**Review by filiphazardous — APPROVED** [review]

**Pull request merged**